outputs.tf

```diff
@@ -35,12 +48,12 @@ output "security_group_ids" {
 
 output "role" {
   description = "Name of AWS IAM Role associated with creating instance"
-  value       = "${join("", aws_iam_role.default.*.name)}"
+  value       = "${join(",", aws_iam_role.default.*.name)}"
 }
 
 output "alarm" {
   description = "CloudWatch Alarm ID"
-  value       = "${join("", aws_cloudwatch_metric_alarm.default.*.id)}"
+  value       = "${join(",", aws_cloudwatch_metric_alarm.default.*.id)}"
```

Comment:
why `,`?

Reply:
We cannot use an empty string or all the IDs will be squashed together if `count > 1`

Reply:
that said, we probably just need to return lists everywhere now.